app/routes/analysis/analysis.py

```diff
@@ -38,7 +39,43 @@ async def zonal_statistics(
 ):
     """Calculate zonal statistics on any registered raster layers in a geostore."""
     geometry = await get_geostore_geometry(geostore_id, geostore_origin)
+    return await _zonal_statics(
+        geometry,
+        sum_layers,
+        group_by,
+        filters,
+        start_date,
+        end_date,
+    )
+
 
+@router.post(
+    "/zonal",
+    response_class=ORJSONResponse,
+    response_model=Response,
+    tags=["Analysis"],
+)
+async def zonal_statistics(
+    request: ZonalAnalysisRequestIn
+):
+    return await _zonal_statics(
+        request.geometry,
+        request.sum,
+        request.group_by,
+        request.filters,
+        request.start_date,
+        request.end_date,
+    )
+
+
+async def _zonal_statics(
+    geometry: Dict[str, Any],
+    sum_layers: List[RasterLayer],
+    group_by: Optional[List[RasterLayer]],
+    filters: Optional[List[RasterLayer]],
```

Comment:
Looking at the signature above again (line 35 and 36).

If default values for group_by and filters is `[]` type is no longer `Optional`. It should be `None`. When using value `[]` you can simply use `List[RasterLayer]`